utils/soldier_builder.py

```
@@ -0,0 +1,16 @@
+from models.soldier import Soldier
+from tests import last_names, soldier
```

Ruff (0.12.2)

`tests.last_names` imported but unused

Remove unused import

(F401)